demisto_sdk/commands/common/constants.py

```diff
@@ -933,6 +942,21 @@ class PB_Status:
     }
 ]
 
+MAX_FETCH_PARAM = {
+        'name': 'max_fetch',
+        'required': False,
+        'type': 0,
+        'defaultvalue': '50'
+    }
+
+FIRST_FETCH_PARAM = {
+    'defaultvalue': '7 days',
```

Comment:
we do not enforce this defaultvalue right? so I do not think it is needed.

Reply:
we don't enforce it, I was debating wether to keep it or not but I think that as an example we should keep it, it might help to prevent future bugs, wdut?

Reply:
better to keep and comment that it is nor validate, just for reference/consideration.